index.html

```diff
@@ -1,82 +1,327 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Mobile Portfolio</title>
-  <link rel="stylesheet" href="style.css">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="css/style.css" rel="stylesheet">
+
+    <title>David Verde</title>
+
 </head>
 
 <body>
-  <header>
-
-    <section class="top-nav">
-
-
-      <a href="https://www.linkedin.com/in/david-verde-3349b114b/e"> <img id="mylogo"
-          src="./Image/logo-david-nuevo-2021.png" alt="logo"> </a>
-      
-      <nav>
-        <label for="menu-toggle">MY FIRST WEB PAGE</label>
-        <input id="menu-toggle" type="checkbox" />
-        <label class='menu-button-container' for="menu-toggle">
-          <span class='menu-button'></span>
-        </label>
-        <ul class="menu">
-          <li>Portfolio</li>
-          <li>About</li>
-          <li>Contact</li>
-        </ul>
-      </nav>
+
+    <header>
+        <nav class="navbar">
+            <a href="https://www.linkedin.com/in/david-verde-3349b114b/e"> <img id="mylogo"
+                    src="./image/logo-david-nuevo-2021.png" alt="logo"> </a>
+
+            <ul class="nav-list">
+                <li class="list-item">
+                    <a href="#"> Portfolio </a>
+                </li>
+                <li class="list-item">
+                    <a href="#"> About </a>
+                </li>
+                <li class="list-item">
+                    <a href="#"> Contact </a>
+                </li>
+            </ul>
+            <div class="menu">
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+            </div>
+        </nav>
+    </header>
+
+    <section class="headlines">
+        <div class="container">
+            <div class="headline-text">
+
+                <h1 class="primary-text">I’m David Verde<br> Glad to see you!</h1>
+                <p>Using a series of utilities, you can create this jumbotron, just like the one in previous versions of
+                    Bootstrap. Check out the examples below for how you can remix and restyle it to your liking.</p>
+                <a href="#" class="btn-connect">
+                    <h6>LET´S CONNECT</h6>
+                </a>
+                <ul class="small_text">
+
+                    <li>
+                        <a href="https://twitter.com/UnyieldingOne">
+                            <img class="size" src="./image/twitter.svg" alt="" /></a>
```

Comment:
- [ ] Your logo image is not displayed, because your using the wrong path for your image, kindly fix this, by changing the path to ``` "../Image/twitter.svg"```